analysers/analyser_osmosis_highway_floating_islands.py

```diff
@@ -110,6 +124,8 @@ class Analyser_Osmosis_Highway_Floating_Islands(Analyser_Osmosis):
 
     def __init__(self, config, logger = None):
         Analyser_Osmosis.__init__(self, config, logger)
+        if not "proj" in self.config.options:
+          return
```

Comment:
Naybe the issue is requiring `highways` without `proj`, but nor directly here ?

Reply:
Well, this is actually because the table `highways` requires `proj` and a.o. opengeofiction doesn't have proj set.
https://github.com/osm-fr/osmose-backend/pull/1554#issuecomment-1245244954

Reply:
Yes, please, remove from here and open a dedicated issue.

Reply:
I'll remove it. Not going to open an issue for such a minor thing, it's only because you asked me in the past to add it that I added it now too ;)

Reply:
Hum, ok, but also bad idea last time. I made the issue #1701.